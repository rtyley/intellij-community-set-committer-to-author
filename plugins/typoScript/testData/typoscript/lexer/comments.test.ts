/*
a comment */
*/

/**
also comment
 */

 /* */
comment   */
  */ ignored text

assign = ment
