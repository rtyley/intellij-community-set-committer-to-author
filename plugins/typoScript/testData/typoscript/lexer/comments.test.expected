C_STYLE_COMMENT ('/*
a comment */
*/')
WHITE_SPACE_WITH_NEW_LINE ('

')
C_STYLE_COMMENT ('/**
also comment
 */')
WHITE_SPACE_WITH_NEW_LINE ('

 ')
C_STYLE_COMMENT ('/* */
comment   */
  */')
IGNORED_TEXT (' ignored text')
WHITE_SPACE_WITH_NEW_LINE ('

')
object path entity ('assign')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
assignment value ('ment')
WHITE_SPACE_WITH_NEW_LINE ('
')